app/services/computacenter_ledger_csv.rb

```diff
@@ -0,0 +1,101 @@
+require 'csv'
+
+class ComputacenterLedgerCsv
+  def initialize(users: nil)
+    @users = users
+  end
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << headers
+
+      ledger_entries.each do |ledger_entry|
+        csv << [
+          ledger_entry.first_name,
+          ledger_entry.last_name,
+          ledger_entry.email,
+          ledger_entry.telephone,
+          ledger_entry.responsible_body,
+          ledger_entry.responsible_body_urn,
+          ledger_entry.cc_sold_to_number,
+          ledger_entry.school,
+          ledger_entry.school_urn,
+          ledger_entry.cc_ship_to_number,
+          ledger_entry.updated_at_timestamp.utc.strftime('%d/%m/%Y'),
+          ledger_entry.updated_at_timestamp.utc.strftime('%R'),
+          ledger_entry.updated_at_timestamp.utc.iso8601,
+          ledger_entry.type_of_update,
+          ledger_entry.original_email,
+        ]
+      end
+    end
+  end
+
+private
+
+  def computacenter_identifier(responsible_body)
+    return nil if responsible_body.nil?
+
+    case responsible_body.type
+    when 'LocalAuthority'
+      "LEA#{responsible_body.gias_id}"
+    when 'Trust'
+      "t#{responsible_body.companies_house_number.to_i}" # QUESTION: is `to_i` needed? is it to remove padded zeros?
```

Comment:
can you confirm this is correct?

Reply:
Yeah it's to remove the leading zero if one's there...